server/prisma/schema.prisma

```diff
@@ -0,0 +1,17 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("MONGODB_URL")
```

Comment:
this is coming from the docker-compose file, and is basically the secret sauce to connecting. If you wanted to connect to a db in prod, you would make sure this ENV var points at a url to mongodb